Action.h

```diff
@@ -6,6 +6,7 @@ htop - Action.h
 Released under the GNU GPLv2+, see the COPYING file
 in the source distribution for its full text.
 */
+#include "config.h" // IWYU pragma: keep
```

Comment:
This should go into `Action.c` and other files using `Action.h`. Otherwise you can't guarantee that this is the first overall included header.

Reply:
Oh okay, I get it. It must be in `c ` to be sure it is the first include.